### ISSUE TITLE
[A11y][SF][Table]: The chevron (>) elements column lack alternate text.

### ISSUE THREAD

**Issue by VidhiNPatel** [UI5/webcomponents-react#7924]
### Describe the bug

Chevron icons are used visually to indicate that the rows are interactive or actionable. This information should also be conveyed to users of screen readers. The cells in the last column receive focus when navigating with the arrow keys; therefore, they should have appropriate alternative text. 
Although the arrow icon SVGs now have aria-label="navigation", the presence of the aria-hidden attribute prevents them from being exposed to assistive technologies. Ensure that the name is exposed properly to screen readers.

Reference: https://ui5.sap.com/#/entity/sap.f.FlexibleColumnLayout/sample/sap.f.sample.FlexibleColumnLayoutWithOneColumnStart

### Reproduction steps

1. Go to https://ui5.github.io/webcomponents-react/v2/?path=/story/data-display-table--with-row-actions 
2. Ensure that the chevron (>) element aria-label name is exposed properly to screen readers.

### Expected Behaviour

Chevron icons are used visually to indicate that the rows are interactive or actionable. This information should also be conveyed to users of screen readers.
Reference: https://ui5.sap.com/#/entity/sap.f.FlexibleColumnLayout/sample/sap.f.sample.FlexibleColumnLayoutWithOneColumnStart

### UI5 Web Components for React Version

2.16

### UI5 Web Components Version

1.16

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
